gulpfile.ts

```diff
@@ -25,6 +25,30 @@ gulp.task("build", () => {
   );
 });
 
+// build and then run coverage
+gulp.task("bnc", () => {
+  runSequence(
+    "clean",
+    "compile",
+    "place-fixture",
+    "browserify",
```

Comment:
There is no more 'browserify'. Please check Sarvesh's changes for a correct merge